.github/ISSUE_TEMPLATE/project-add.yml

```diff
@@ -1,7 +1,7 @@
 name: 🌱 Project Addition
 description: Add a new project to the list
 title: "data: project addition by "
-labels: ["🌱 chore: projects addition", "👍  status: approved", "gssoc23", "level1"]
+labels: ["🌱 chore: projects addition", "👍  status: approved", "gssoc23", "🟨 level 1"]
```

Comment:
``` suggestion
labels: ["🌱 chore: projects addition", "👍  status: approved", "gssoc23", "level1"]
```

Reply:
sir, I already made a PR for data addition which was merged yesterday

Reply:
ok, so could you create a separate & fresh PR? it will be good

Reply:
now I'm closing this PR